tests/base/src/test/java/org/wildfly/security/ssl/SSLAuthenticationTest.java

```diff
@@ -17,12 +17,6 @@
  */
 package org.wildfly.security.ssl;
 
-import static org.junit.Assert.assertEquals;
```

Comment:
Just a minor comment here, we can leave these static imports here.

Reply:
Fixed, thanks!